src/Components/benchmarkapps/Wasm.Performance/TestApp/Pages/OrgChart.razor

```diff
@@ -17,8 +17,8 @@
 
 @code
 {
-    int depth = 2;
-    int subs = 5;
+    int depth = 1;
```

Comment:
Intentional. We spend too much time rendering this, which makes testing navigating to this component less useful.

Reply:
Isn't this also used in a rendering perf benchmark though?

If so maybe we should have two copies of it: the existing one that does very intensive rendering, and a new one using your new consts that's intended for use in stress tests.

Overall, perhaps we should have a clearer split between perf and stress tests, for example separate named directories.